<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-0AGRRL1R" failures="2" tests="10" name="Test" time="4.627" errors="0" timestamp="2023-08-30T21:09:52 ICT">
  <testcase classname="testcase.Example1Test" name="T01_StatusCodeTest" time="0.014"/>
  <testcase classname="testcase.Example1Test" name="T02_IdChecked" time="0.002"/>
  <testcase classname="testcase.Example1Test" name="T03_EmailChecked" time="0.002"/>
  <testcase classname="testcase.Example1Test" name="T04_EmailChecked" time="0.002"/>
  <testcase classname="testcase.Example1Test" name="T05_EmailChecked" time="0.001"/>
  <testcase classname="testcase.Example1Test" name="T06_AvatarChecked" time="0.002"/>
  <testcase classname="testcase.Example1Test" name="T07_Compare_id" time="0.755"/>
  <testcase classname="testcase.Example2Test" name="T01_StatusCodeTest" time="0.001"/>
  <testcase classname="testcase.Example2Test" name="T02_MessageChecked" time="0.003">
    <failure type="java.lang.AssertionError" message="message field check Failed! expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: message field check Failed! expected [false] but found [true]
at testcase.Example2Test.T02_MessageChecked(Example2Test.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- T02_MessageChecked -->
  <testcase classname="testcase.Example2Test" name="T03_MessageContentCheck" time="0.04">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcase.Example2Test.T03_MessageContentCheck(Example2Test.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- T03_MessageContentCheck -->
</testsuite> <!-- Test -->
